flang/docs/Intrinsics.md

```diff
@@ -700,7 +700,7 @@ IBCHNG, ISHA, ISHC, ISHL, IXOR
 IARG, IARGC, NARGS, NUMARG
 BADDRESS, IADDR
 CACHESIZE, EOF, FP_CLASS, INT_PTR_KIND, ISNAN, LOC
-MALLOC
+MALLOC, FREE
```

Comment:
Note: the docs say that MALLOC is an extension we already support, but it isn't actually supported at the moment.